tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -788,78 +788,6 @@ def test_negative_update_with_name(self, new_name, content_view, module_target_s
                 {'content-view-id': content_view['id'], 'name': new_name}
             )
 
-    @pytest.mark.tier1
-    def test_negative_update_with_same_name(self, module_org, content_view, module_target_sat):
```

Comment:
I believe this deletion must be due to low severity of test, don't see any qe-recommendation in the sheet.

Reply:
marked as QE delete